modules/openxr/extensions/openxr_composition_layer_extension.cpp

```diff
@@ -113,6 +138,37 @@ bool OpenXRCompositionLayerExtension::is_available(XrStructureType p_which) {
 	}
 }
 
+#ifdef ANDROID_ENABLED
+bool OpenXRCompositionLayerExtension::create_android_surface_swapchain(XrSwapchainCreateInfo *p_info, XrSwapchain *r_swapchain, jobject *r_surface) {
+	if (android_surface_ext_available) {
+		OpenXRAPI *openxr_api = OpenXRAPI::get_singleton();
+		ERR_FAIL_NULL_V(openxr_api, false);
+
+		// @todo We need a way to add to the next pointer chain.
```

Comment:
Do we have a capability that depends on chaining or is it for future proofing?

Reply:
Specifically to support `XR_FB_android_surface_swapchain_create`, which we'll want to be able to add from the vendor extension